pkg/features/features.go

```diff
@@ -101,4 +101,9 @@ const (
 	// owner: @petersutter @xrstf
 	// alpha: v1.28.0
 	RotateSSHKeypairOnMaintenance featuregate.Feature = "RotateSSHKeypairOnMaintenance"
+
+	// DenyInvalidExtensionResources causes the seed-admission-controller to deny invalid extension resources (instead of just logging validation errors).
+	// owner: @vanjiii
+	// alpha: v1.31.0
+	DenyInvalidExtensionResources featuregate.Feature = "DenyInvalidExtensionResources"
```

Comment:
Can you add the feature gate to `doc/deployment/feature_gates.md` as well?

Reply:
Added